pkg/graph/schema.graphql

```diff
@@ -54,6 +54,66 @@ type CedarSystem {
   versionId: String
 }
 
+"""
+BusinessOwnerInformation contains information about the business owner for a CEDAR system
+"""
+type CedarBusinessOwnerInformation {
+	beneficiaryAddressPurpose: [String!]
```

Comment:
I'm inclined to make this array non-nullable; testing a couple of systems in Postman, CEDAR seems to respond with an empty array if there's no data, rather than a JSON `null`. (I tested `/system/detail/326-7-0`, `/system/detail/326-9-0`, and `/system/detail/326-2403-0`).